### PR TITLE
Change iOS implementation to Material design Snackbar

### PR DESCRIPTION
**Pull request by bartsg**

### CODE REVIEW

**Review by milanosie — APPROVED** [review]

File: src/platforms/ios/Podfile [hunk 1]
```diff
@@ -1 +1 @@
-pod 'SSSnackbar', :git => 'https://github.com/EddyVerbruggen/SSSnackbar.git', :tag => '0.1.2.4'
\ No newline at end of file
+pod 'MaterialComponents/Snackbar'
```
Comment (dennis-montana) [inline src/platforms/ios/Podfile]: I think it's a good idea to add a version number so the plugin will not break when there is a new version with breaking changes.

File: README.md [hunk 2]
```diff
@@ -16,20 +12,14 @@ Use the Material Design Snackbar in your {N} app, iOS uses [SSSnackbar Cocoapod]
...
-### NativeScript 3x-
-
-`tns plugin add nativescript-snackbar@2.0.1` or just pin `nativescript-snackbar: 2.0.1` in your app's package.json - to avoid any build conflicts is why the latest has been bumped a major.
-
-Requires Xcode 9+ to build for iOS, so update Xcode if needed.
+`tns plugin add @essent/nativescript-snackbar`
 
 ## Usage
 
 ### TS
```
Comment (dennis-montana) [inline README.md]: Can you remove this title?

**Review by dennis-montana — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (bartsg) [reply README.md]: Done

File: src/platforms/ios/Podfile (hunk 1 above)
Comment (bartsg) [reply src/platforms/ios/Podfile]: It's locked now.

**Review by dennis-montana — APPROVED** [review]

**Pull request merged**